App.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import LoginPage from './LoginPage';
+
+function App() {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/login" component={LoginPage} />
+        <Route exact path="/home">
```

Comment:
There seems to be a missing component for the '/home' route. This could lead to an empty route when navigating to '/home'. Consider adding a component for this route.